### PR TITLE
[FSM] Nice to have

### PR DESCRIPTION
**Pull request by sdegueldre**
Task-2075174

related to https://github.com/odoo/enterprise/pull/5763

### CODE REVIEW

File: addons/project/views/project_views.xml [hunk 1]
```diff
@@ -165,11 +165,6 @@
                                     </div>
                                 </div>
                             </div>
-                            <group name="misc">
```
Comment (jem-odoo) [inline addons/project/views/project_views.xml]: Can you check with AJU, I'm not ok with this point (functionnal: how do we set a calendar on the project ? O_0)

File: addons/project/models/project.py [hunk 2]
```diff
@@ -516,6 +516,7 @@ def _read_group_stage_ids(self, stages, domain, order):
         string='Customer',
         default=lambda self: self._get_default_partner(),
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    commercial_partner_id = fields.Many2one('res.partner', related='partner_id.commercial_partner_id')
```
Comment (jem-odoo) [inline addons/project/models/project.py]: Always show commercial partner on tasks: the commit looks good but we are too close to the release to add field, so sorry but can you drop it ? :s

File: addons/project/views/project_views.xml (hunk 1 above)
Comment (jem-odoo) [reply addons/project/views/project_views.xml]: Ok, she is ok to remove this commit.

**Comment by sdegueldre** [comment]
All the changes should be accounted for, please check that everything looks good :)

**Comment by jem-odoo** [comment]
@robodoo r+

File: addons/project/models/project.py [hunk 3]
```diff
@@ -216,7 +216,7 @@ def _get_default_favorite_user_ids(self):
 
     # rating fields
     rating_request_deadline = fields.Datetime(compute='_compute_rating_request_deadline', store=True)
-    rating_status = fields.Selection([('stage', 'Rating when changing stage'), ('periodic', 'Periodical Rating'), ('no','No rating')], 'Customer(s) Ratings', help="How to get customer feedback?\n"
+    rating_status = fields.Selection([('stage', 'Rating when changing stage'), ('periodic', 'Periodical Rating'), ('no','No rating')], 'Customer Ratings', help="How to get customer feedback?\n"
```
Comment (mart-e) [inline addons/project/models/project.py]: please update the .pot file too when changing translatable content

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**